src/EditorFeatures/CSharpTest2/Recommendations/StackAllocKeywordRecommenderTests.cs

```diff
@@ -124,12 +90,99 @@ unsafe static void Main(string[] args)
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         public async Task TestInsideForStatementVarDecl3()
         {
-            await VerifyAbsenceAsync(
+            await VerifyKeywordAsync(
 @"class C
 {
     unsafe static void Main(string[] args)
     {
         for (string i = $$");
         }
+
+        [WorkItem(23584, "https://github.com/dotnet/roslyn/issues/23584")]
```

Comment:
something is super odd about thsi dif.. it looks like a test method is inside test code.

Reply:
Test string ends on line 98. Curly bracket on line 99 closes the test method :)

---
In reply to: [155677980](https://github.com/dotnet/roslyn/pull/23655#discussion_r155677980) [](ancestors = 155677980)